CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 2.3
+
+- Added health methods (get_notifier_state, disable_notifications, enable_notifications) to prevent<br/>
```

Comment:
1. Add link to docs: https://moira.readthedocs.io/en/latest/user_guide/selfstate.html
2. ` to protect end user from false NODATA.` instead of `to prevent mass spam mailings in cases when Graphite metrics receivers are degraded`